src/features/import/hooks/useImportID.ts

```diff
@@ -0,0 +1,57 @@
+import { importIDUpdate } from '../store';
+import { Column, ColumnKind, ImportID } from '../utils/types';
+import { useAppDispatch, useAppSelector } from 'core/hooks';
+
+export default function useImportID() {
+  const dispatch = useAppDispatch();
+
+  const importID = useAppSelector((state) => state.import.importID);
+
+  const handleColumnDeselection = (
+    column: Column,
+    currentImportID: string | null,
+    onChange: (newColumn: Column) => void
+  ): void => {
+    resetImportIDIfNeeded(column, currentImportID);
```

Comment:
The name of this function makes me think that this is logic that should be in the store, rather than in a hook.